[Feature] Add `created_at` and `updated at` fields (#58)
fivetran/dbt_quickbooks_source: CHANGELOG.md

> # dbt_quickbooks_source v0.11.1

Author: Updated

# dbt_quickbooks_source v0.12.0
[PR #58](https://github.com/fivetran/dbt_quickbooks_source/pull/57) introduces the following updates:

## Breaking Changes

> - We introduced these pre-existing timestamp fields from these sources to their equivalent staging models to better analyze real-time transaction data: 

Reviewer: I would list the field and the exact staging models they were added to instead of requiring the user to have to match them up on their own. 

```suggestion
- Introduced the following timestamp fields to the listed staging models to better analyze real-time transaction data:
```

Author: @fivetran-catfritz Listing all the staging models would get a bit wordy, so I did a slight tweak to your suggestion. Let me know your thoughts.

Reviewer: True--this makes sense!

  - `created_at`: `bill`, `bill_payment`, `credit_memo`, `invoice`, `payment`, `transfer`
  - `updated_at`: `bill`, `bill_payment`, `credit_memo`, `deposit`, `invoice`, `journal_entry`, `payment`, `purchase`, `refund_receipt`, `sales_receipt`, `transfer`, `vendor_credit`

> - - These new fields are incorporated in the `dbt_quickbooks` package's `quickbooks__general_ledger` model via the `*_double_entry` intermediate models. You can learn more about these changes in the [v0.17.0 release of the `dbt_quickbooks` package](https://github.com/fivetran/dbt_quickbooks/releases/tag/v0.17.0).

Reviewer: Just noticed the double dash.
```suggestion
- These new fields are incorporated in the `dbt_quickbooks` package's `quickbooks__general_ledger` model via the `*_double_entry` intermediate models. You can learn more about these changes in the [v0.17.0 release of the `dbt_quickbooks` package](https://github.com/fivetran/dbt_quickbooks/releases/tag/v0.17.0).
```

> - These new fields are then being brought in downstream in the `dbt_quickbooks` package in the `quickbooks__general_ledger` model via the double entry transaction intermediate models. You can learn more about these changes in the [v0.17.0 release of the `dbt_quickbooks` package](https://github.com/fivetran/dbt_quickbooks/releases/tag/v0.17.0). 

Reviewer: ```suggestion
- These new fields are incorporated in the `dbt_quickbooks` package's `quickbooks__general_ledger` model via the `*_double_entry` intermediate models. You can learn more about these changes in the [v0.17.0 release of the `dbt_quickbooks` package](https://github.com/fivetran/dbt_quickbooks/releases/tag/v0.17.0). 
```

> - As this introduces new columns to our staging models and changes our schema, this will be a breaking change. 

Reviewer: ```suggestion
- As this introduces new columns to our staging models and changes our schema, this is breaking change. 
```
